src/first_steps/getting_rizin.md

```diff
@@ -1,6 +1,6 @@
 ## Downloading rizin
 
-You can get rizin from the website, [http://rizin.org](http://rizin.org),
+You can get rizin from the website, [http://rizin.re](http://rizin.re),
```

Comment:
That's currently untrue

Reply:
So, I'll get it back to `rizin.org`. That's what you meant, right?